### PR TITLE
add runner install script for windows image

### PR DESCRIPTION
**Pull request by pshepherd10**
# Description
Read [here](https://github.com/actions/runner-images/pull/8474) for more details. Adding the install script for the runner package for windows images.

## Check list
- [x] Related issue / work item is attached
- [x] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/win/scripts/Installers/Install-Runner.ps1 [hunk 1]
```diff
@@ -0,0 +1,4 @@
+$runnerDownloadUrl = Get-GitHubPackageDownloadUrl -RepoOwner "Actions" -RepoName "runner" -BinaryName "actions-runner-linux-x64" -Version "latest" -UrlFilter "*{BinaryName}-{Version}.zip"
+$fileName = runnerDownloadUrl.Substring($zip.LastIndexOf("/") + 1)
+New-Item -Path "C:\ProgramData\runner" -ItemType Directory
+$kotlinInstallerPath = Start-DownloadWithRetry -Url $runnerDownloadUrl -Name $fileName -DownloadPath "C:\ProgramData\runner"
```
Comment (mikhailkoliada) [inline images/win/scripts/Installers/Install-Runner.ps1:4]: lets introduce a pester test in order to make sure that the target directory exists and not empty

**Comment by mikhailkoliada** [comment]
could you please rebase?

**Comment by pshepherd10** [comment]
@mikhailkoliada this is updated as well, sorry for the delay. I'm waiting to confirm the tests on the other Pr work before adding it here.

**Comment by pshepherd10** [comment]
@mikhailkoliada ok added the test and I think It's ready for image generation

File: images/win/scripts/Tests/RunnerCache.tests.ps1 [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Describe "RunnerCache" {
+    Context "runner cache directory not empty" {
+        It "/opt/runner-cache not empty" {
```
Comment (erik-bershel) [inline images/win/scripts/Tests/RunnerCache.tests.ps1:3]:
```suggestion
        It "C:\ProgramData\runner" {
```

File: images/win/scripts/Tests/RunnerCache.tests.ps1 [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+            (Get-ChildItem -Path "/opt/runner-cache/*.tar.gz" -Recurse).Count | Should -BeGreaterThan 0
```
Comment (erik-bershel) [inline images/win/scripts/Tests/RunnerCache.tests.ps1:4]:
```suggestion
            (Get-ChildItem -Path "C:\ProgramData\runner\*zip" -Recurse).Count | Should -BeGreaterThan 0
```

**Review by mikhailkoliada — APPROVED** [review]

**Pull request merged**